src/sentry/apidocs/parameters.py

```diff
@@ -8,7 +10,7 @@
 
 # drf-spectacular doesn't support a list type in it's OpenApiTypes, so we manually build
 # a typed list using this workaround
-def build_typed_list(type: OpenApiTypes):
+def build_typed_list(type: Any):
```

Comment:
You can't really put any type though right? Like does it work with serialized object? What types do we want to support that is not in OpenApiTypes?

Reply:
basically any base python type. `build_typed_list` calls `build_basic_type` which does type checking dynamically for us. If it fails to register the type, it gives a warning that the argument type fails. which also fails any build command

Reply:
Can you add that in the comment? It sounds good if the build fails.